src/export_data.py

```diff
@@ -0,0 +1,122 @@
+"""Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import argparse
+import ee
+from utils import TaskManager, SampleExporter, stack_bands_from_imagery
```

Comment:
Can we collapse these all into one "from utils import....."?

Reply:
I was avoiding long lines